docs/application/dotnet/guides/opentk/work-with-skiasharp.md

```diff
@@ -0,0 +1,183 @@
+# Work with SkiaSharp
+
+SkiaSharp is a cross-platform 2D graphics API for .NET platforms based on Google's Skia Graphics Library. 
+It provides a comprehensive 2D API that can be used across mobile, server and desktop models to render images.
+
+In OpenTK application, you can use SkiaSharp to draw geometries, texts or images. For more information about the SkiaSharp APIs, please reference to [SkiaSharp API document](https://docs.microsoft.com/en-us/dotnet/api/skiasharp?view=skiasharp-1.60.3).
+
+To draw something with SkiaSharp in OpenTK application, it following some general guidelines:
+
+-   Allocate a memory block as a canvas for SkiaSharp.
+-   Create SKCanvas on this memory block and render text/image you want on this SKCanvas with SkiaSharp.
+-   Create 2D texture from this memory block.
+-   Draw 2D texture with graphics APIs of OpenTK.
+
+## Allocate memory block
+
+Allocate a specific size memory block. The size of memory block must be same with the size of the canvas you want.
```

Comment:
Please change to:
Allocate a specific size of the memory block. The size of memory block must be same as the size of the canvas as you want.